app/src/main/java/com/mikhailovskii/weatherandroid/util/Constants.kt

```diff
@@ -0,0 +1,15 @@
+package com.mikhailovskii.weatherandroid.util
+
+const val HALF_SCREEN = 0.5
+
+const val QUARTER_SCREEN = 0.25
+
+const val BASE_MARGIN = 16
+
+const val PERMISSION_CODE = 100
+
+const val DIFFERENCE_KELVIN_CELSIUS = 273
+
+const val ZOOM_LEVEL = 10.0f
+
+const val USERS_COLLECTION = "users"
```

Comment:
@SergeiMikhailovskii do you really need these constants to be accessed globally? I suppose it makes sense to logically separate them into different containers. Some of them could be `private` for specific classes (eg. `DIFFERENCE_KELVIN_CELSIUS` makes sense only for `WeatherCalculations`)